### DISCUSSION TITLE
Add series from favorites from TVDB

### DISCUSSION THREAD

**Discussion by ciltocruz** [pymedusa/Medusa#8029] [Ideas & Feature requests]
Hello!

I am trying Medusa and I found it interesting to ask for this.
I have +100 series added to favorites on tvdb. Transferring these favorites to Medusa would be a hard job.

Could you add the option to connect with API token to TVDB (of each user) and retrieve by API the favorites list to import it into Medusa?

Thank you!

**Comment by medariox** [comment]
I think you got the wrong software, this is Medusa.

**Comment by ciltocruz** [comment]
Ok. I change the name.
The question is same. 
Please, reopen ticket. 
Thanks.
